### ISSUE TITLE
PatientFlatWriter.closeAndKeepAlive should call _writer.flush()

### ISSUE THREAD

**Issue by criderp** [imsweb/naaccr-xml#105]
I think that is the intention of that method.

**Comment by depryf** [comment]
Agreed. I guess it works in the PatientXmlWriter because that one calls "endNode", but that's not exactly the same thing, and so I guess I will add the flush to both.

**Comment by criderp** [comment]
I should mention that this is not holding anything up in DMS.
